File: Codigo/ArenaGestor/ArenaGestor.APIContracts/Ticket/TicketSnackBuyDto.cs
using ArenaGestor.APIContracts.Snacks;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ArenaGestor.APIContracts.Ticket
{
    public class TicketSnackBuyDto
    {
        public List<TicketGetTicketResultDto> TicketDto { get; set; }
        public List<SnackBuyLineDto> SnacksDto { get; set; }
    }
}

File: Codigo/ArenaGestorFront/src/app/models/Tickets/TicketSnackBuyDto.ts
import { SnackBuyLineDto } from "../Snacks/SnackBuyLineDto";
import { TicketGetTicketResultDto } from "./TicketGetTicketResultDto";

export class TicketSnackBuyDto {
    ticketDto: Array<TicketGetTicketResultDto> = new Array(); 
    snacksDto: Array<SnackBuyLineDto> = new Array();
}